app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(index edit update destroy)
```

cơ mà quá nửa actions (7 action chuẩn restful), nên tìm heieru về `except` đi em nhé thay vì only